src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -168,15 +174,13 @@ def _initialise_aperture(self):
         finally:
             self._is_moving = False
 
-    def _move_zero(self):
-        self.logger.debug('Setting focus encoder zero point')
-        if self._position != 0:
-            self._is_moving = True
-            try:
-                # Set focuser to 0 position
-                self._send_command('M0')
-                self.logger.debug('Focuser has been set to encoder position 0')
-            finally:
-                self._is_moving = False
-        else:
-            self.logger.debug('Focuser already at encoder position 0')
+    def _calibrate_near_stop(self):
```

Comment:
The base class and the Birger have a `_move_zero` method. This is basically moving to what it thinks the zero point is, so it's probably better to keep the same name for consistency.

```suggestion
    def _move_zero(self):
```